services/slam/slam.go

```diff
@@ -119,6 +122,19 @@ func GetInternalStateFull(ctx context.Context, slamSvc Service) ([]byte, error)
 	return helperConcatenateChunksToFull(callback)
 }
 
+// Limits gets the extents of the SLAM map
+func Limits(ctx context.Context, slamSvc Service) ([]referenceframe.Limit, error) {
```

Comment:
[q] Is there a ticket for adding this function?

Reply:
[q] why is this only returning the limits in the XZ plane?

Reply:
`[]referenceframe.Limit` doesn't seem very type-safe to me. In order for the caller to get a value they can do something with it needs to specifically be a `[]referenceframe.Limit` of size 2 right?

Is there a reason why we are not using an array of size 2 or a struct that makes it explicit that this is measuring the min & max dimensions in the XZ plane?

Something like:
```
type XYZLimit struct {
X referenceframe.Limit
Y referenceframe.Limit
Z referenceframe.Limit
}
```

Reply:
Output of convo:
I think that this function should either exist in the package of the service that uses slam given that this function as written doesn't allow a user to access or perscribe the point cloud who's limits were analyzed. If this function ends up being duplicated in more than one other package I think we can consider adding it to the slam.go.

Reply:
Per our discussion this is not a useful thing to add to the slam package.  It has been removed

Reply:
Thanks for summarizing our convo!  Removed the function from the slam package